<?xml version="1.0"?>

<launch>
    <arg name="respawn_node" default="false"/>
    <arg name="node_name" default="comm_bridge" />
    <arg name="local_server" default="false"/>
    <arg name="params_path" default="$(find ros_kafka_connector)/config/params.yaml"/>

    <node pkg="ros_kafka_connector" type="comm_bridge.py" name="$(arg node_name)" output="screen" respawn="$(arg respawn_node)">
        <rosparam command="load" file="$(arg params_path)"  subst_value="true"/>
        <rosparam param="local_server">$(arg local_server)</rosparam>
    </node>

</launch>
